### PR TITLE
Adding home image background, organizing general styling

### PR DESCRIPTION
**Pull request by MarianaSouza**

### CODE REVIEW

File: src/components/HowItWorks/HowItWorks.scss [hunk 1]
```diff
@@ -61,30 +61,11 @@ section.how-it-works-mobile {
...
-      }
-    }
-  }
-  nav.navbar {
-    padding: 20px 60px;
-  }
 }
 @media (max-width: 768px) {
   &:root {
     font-size: 16px;
```
Comment (emersonmellado) [inline src/components/HowItWorks/HowItWorks.scss:67]: Hmm, this looks fishy. It looks like a App.scss media query not something that the howitworks module require. PLease take a look :)

File: src/components/Navbar/Navbar.scss [hunk 2]
```diff
@@ -1,4 +1,8 @@
-@import "../../css/common.scss";
+@import "../../css/variables.scss";
+
+.navbar-ams{
+  padding-top: 2.2rem!important;
```
Comment (emersonmellado) [inline src/components/Navbar/Navbar.scss:4]: I think we probably don't need the !important here now that we found the css duplication.

File: src/components/Navbar/Navbar.scss [hunk 3]
```diff
@@ -22,7 +26,7 @@ a.nav-link {
   &:focus {
     outline: none;
   }
-  span {
+  span.icon-bar {
```
Comment (emersonmellado) [inline src/components/Navbar/Navbar.scss:29]: I like that, more specific the better!

File: src/components/Search/Search.scss [hunk 4]
```diff
@@ -1,14 +1,14 @@
-@import "./../../css/common.scss";
+@import "./../../css/variables.scss";
 
 .support-search {
```
Comment (emersonmellado) [inline src/components/Search/Search.scss:3]: I don't like this name, the component is called Search and the css class support-search. It feels a bit confusing. What about renaming it to be just search?

Comment (MarianaSouza) [reply src/components/Search/Search.scss:3]: Can it be search?

Comment (MarianaSouza) [reply src/components/Search/Search.scss:3]: The class was renamed

**Pull request merged**